vars/parallelDockerUpdatecli.groovy

```diff
@@ -35,16 +35,14 @@ def call(userConfig = [:]) {
       }
     },
     'updatecli': {
-      withCredentials([string(credentialsId: finalConfig.credentialsId,variable: 'UPDATECLI_GITHUB_TOKEN')]) {
-        // Merging the 2 maps - https://blog.mrhaki.com/2010/04/groovy-goodness-adding-maps-to-map_21.html
-        Map updatecliConfig = finalConfig.updatecliConfig << [action: 'diff']
+      // Merging the 2 maps - https://blog.mrhaki.com/2010/04/groovy-goodness-adding-maps-to-map_21.html
+      Map updatecliConfig = finalConfig.updatecliConfig << [action: 'diff', credentialsId: finalConfig.credentialsId]
 
+      updatecli(updatecliConfig)
+      if (env.BRANCH_IS_PRIMARY) {
+        // Merging the 2 maps - https://blog.mrhaki.com/2010/04/groovy-goodness-adding-maps-to-map_21.html
+        updatecliConfig = finalConfig.updatecliConfig << [action: 'apply', cronTriggerExpression: finalConfig.updatecliApplyCronTriggerExpression]
```

Comment:
```suggestion
        updatecliConfig = finalConfig.updatecliConfig << [action: 'apply', cronTriggerExpression: finalConfig.updatecliApplyCronTriggerExpression, credentialsId: finalConfig.credentialsId]
```

Reply:
This config part is already included in finalConfig L39, no need to include it again.

Reply:
Or do I miss something?

Reply:
I confirm that it is not part of final config. 

- [L39](https://github.com/jenkins-infra/pipeline-library/pull/320/files#diff-fd1e84d84679aaa1bc63579323628681c111e48b2aa5a90c765a7f153cdcff79R39) defines a new variable named `updatecliConfig` of type `Map` which is the result of merging finalConfig with the map `[action: 'diff', credentialsId: finalConfig.credentialsId]`.
- The current line pointed by this thread assign a new value to the variable `updatecliConfig`, which is the merge of `finalConfig` with  `[action: 'apply', cronTriggerExpression: finalConfig.updatecliApplyCronTriggerExpression, credentialsId: finalConfig.credentialsId]`. But it's a different one.

Reply:
My bad 😞